handler/endpoint.go

```diff
@@ -163,6 +164,20 @@ func (e *Endpoint) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 			serveErr = errors.Server.With(err)
 			log.WithError(err).Error()
 		}
+
+		content, _, _ := e.opts.Context.PartialContent(config.Endpoint{}.Schema(true))
+		if attr, ok := content.Attributes["set_response_status"]; ok {
+			if evalErr, statusCode := eval.ApplyResponseStatus(evalContext, attr, nil); statusCode > 0 {
+				if serr, k := serveErr.(*errors.Error); k {
+					serveErr = serr.Status(statusCode)
+				} else {
+					serveErr = errors.Server.With(serveErr).Status(statusCode)
+				}
+			} else if evalErr != nil {
+				e.log.WithError(errors.Evaluation.With(evalErr))
```

Comment:
An Evaluation error seems to be not entirely correct. In some cases the method returns `errors.Configuration` error.